### ISSUE TITLE
When using Whisper's auto-detected language, insert that language into the Cocina

### ISSUE THREAD

**Issue by andrewjbtw** [sul-dlss/speech-to-text#45]
When testing out captioning for Bengali language video, Whisper output text in what appears to be Bengali. (After checking with the curator for South Asian materials, it appears that we do not have anyone on staff who can read or speak Bengali.)

However, we are not yet applying a language tag to these caption files, which results in the display showing the language as English (the default). If we can get the language from Whisper, then we have a place to put it in the Cocina for the VTT file, like so:

```json
"type": "https://cocina.sul.stanford.edu/models/file",
              "externalIdentifier": "https://cocina.sul.stanford.edu/file/9ae07267-1b89-40c3-a6b2-ad265894ab66",
              "label": "qf378nj5000_spa_cap.vtt",
              "filename": "qf378nj5000_spa_cap.vtt",
              "size": 54775,
              "version": 17,
              "hasMimeType": "text/vtt",
              "languageTag": "es",
              "use": "caption",
```
Note that even though "spa" (for Spanish) is in the filename of the vtt in that example, it's the `languageTag` field that makes the difference to the display.

Screenshot of current display:

<img width="678" alt="Screenshot 2024-11-15 at 10 55 08 AM" src="https://github.com/user-attachments/assets/d832f901-2835-4221-84ab-a112a0a27654">

**Comment by jmartin-sul** [comment]
note: there might be work in this repo, to return the caption language in the "done" message that's queued when whisper completes, as well as in common-accessioning, to update the cocina accordingly.  totally fine to split the common-accessioning work into a separate ticket that's blocked by this one, for whoever picks this one up.

**Comment by peetucket** [comment]
Possible approach:

1. Adjust `StageFiles` class for speech to text to retrieve json files from S3 in addition to txt and vtt.
2. Adjust `CocinaUpdater` class for speech to text to parse the json file, find the language and add to the structural for that file (while otherwise ignoring the .json file and not adding it to cocina).  

These changes are purely to common-accessioning and do not require changing any messaging or whipser stuff.
